modules/solid_mechanics/neml2/src/LAROMANCE6DInterpolation.cxx

```diff
@@ -0,0 +1,374 @@
+#include "LAROMANCE6DInterpolation.h"
+#include <fstream>
+#include <initializer_list>
+#include "neml2/misc/math.h"
+
+namespace neml2
+{
+register_NEML2_object(LAROMANCE6DInterpolation);
+
+OptionSet
+LAROMANCE6DInterpolation::expected_options()
+{
+  auto options = Model::expected_options();
+  options.doc() = "Multilinear interpolation over six dimensions (von_mises_stress, temperature, "
+                  "equivalent_plastic_strain, cell_dd_density, wall_dd_density, env_factor)";
+  // Model inputs
+  options.set<VariableName>("equivalent_plastic_strain") = VariableName("state", "ep");
+  options.set<VariableName>("von_mises_stress") = VariableName("state", "s");
+
+  options.set<VariableName>("cell_dd_density") = VariableName("old_state", "cell_dd");
+  options.set<VariableName>("wall_dd_density") = VariableName("old_state", "wall_dd");
+
+  options.set<VariableName>("temperature") = VariableName("forces", "T");
+  options.set<VariableName>("env_factor") = VariableName("forces", "env_fac");
+  // Model Outputs
+  options.set_output("output_rate");
+  // JSON
+  options.set<std::string>("model_file_name");
+  options.set<std::string>("model_file_variable_name");
+
+  options.set<bool>("_use_AD_first_derivative") = true;
+  options.set<bool>("_use_AD_second_derivative") = true;
```

Comment:
```suggestion
```
The API for requesting AD has changed. Take a look at `NEML2TestModel.cxx`

Reply:
@dschwen helped me get this to work but I had to turn off jit.